docs/ops/sort/NonMaxSuppression_5.md

```diff
@@ -0,0 +1,119 @@
+
+This algorithm is applied independently to each class of each batch element. The total number of output boxes for each
+class must not exceed `max_output_boxes_per_class`.
+
+**Attributes**:
+
+* *box_encoding*
+
+  * **Description**: *box_encoding* specifies the format of boxes data encoding.
+  * **Range of values**: "corner" or "center"
+    * *corner* - the box data is supplied as `[y1, x1, y2, x2]` where `(y1, x1)` and `(y2, x2)` are the coordinates of any diagonal pair of box corners.
+    * *center* - the box data is supplied as `[x_center, y_center, width, height]`.
+  * **Type**: string
+  * **Default value**: "corner"
+  * **Required**: *no*
+
+* *sort_result_descending*
+
+  * **Description**: *sort_result_descending* is a flag that specifies whenever it is necessary to sort selected boxes across batches or not.
+  * **Range of values**: True of False
+    * *True* - sort selected boxes across batches.
+    * *False* - do not sort selected boxes across batches (boxes are sorted per class).
+  * **Type**: boolean
+  * **Default value**: True
+  * **Required**: *no*
+
+* *output_type*
+
+  * **Description**: the output tensor type
+  * **Range of values**: "i64" or "i32"
+  * **Type**: string
+  * **Default value**: "i64"
+  * **Required**: *No*
+
+**Inputs**:
+
+*   **1**: `boxes` - tensor of type *T* and shape `[num_batches, num_boxes, 4]` with box coordinates. Required.
+
+*   **2**: `scores` - tensor of type *T* and shape `[num_batches, num_classes, num_boxes]` with box scores. Required.
+
+*   **3**: `max_output_boxes_per_class` - scalar or 1D tensor with 1 element of type *T_MAX_BOXES* specifying maximum number of boxes to be selected per class. Optional with default value 0 meaning select no boxes.
+
+*   **4**: `iou_threshold` - scalar or 1D tensor with 1 element of type *T_THRESHOLDS* specifying intersection over union threshold. Optional with default value 0 meaning keep all boxes.
+
+*   **5**: `score_threshold` - scalar or 1D tensor with 1 element of type *T_THRESHOLDS* specifying minimum score to consider box for the processing. Optional with default value 0.
+
+*   **6**:  `soft_nms_sigma` - scalar or 1D tensor with 1 element of type *T_THRESHOLDS* specifying the sigma parameter for Soft-NMS; see [Bodla et al](https://arxiv.org/abs/1704.04503.pdf). Optional with default value 0.
+
+**Outputs**:
+
+*   **1**: `selected_indices` - tensor of type *T_IND* and shape `[number of selected boxes, 3]` containing information about selected boxes as triplets `[batch_index, class_index, box_index]`.
+
+*   **2**: `selected_scores` -  tensor of type *T_THRESHOLDS* and shape `[number of selected boxes, 3]` containing information about scores for each selected box as triplets `[batch_index, class_index, box_score]`.
+
+*   **3**: `valid_outputs` -  scalar of type *T_IND* representing the total number of selected boxes. Optional.
```

Comment:
Change to 1D tensor with 1 element (not a scalar). It is more plugin friendly.

Reply:
Done.